README.md

```diff
@@ -45,3 +45,32 @@ Just a word for the localization:
 The localized entries are on the _config/localization_ folder. Each entries have a key. To use one of this entry, just use ```{ loc: key.subkey }``` instead of a string. (For the titles and contents).
 
 If you want complex content, just create a TWIG template on the _views/contents_ and use ```{ content: templateName }``` instead of a string.
+
+## How to build the OnBoarding module
+
+The assets can already be found in this repo, but if you need to update the files, you will have to run webpack again.
+This module uses Webpack 4, so make sure you have all packages and the cli tool installed first:
+```shell
+$ npm i
+$ npm i -g webpack-cli 
```

Comment:
if webpack-cli is needed, it must be include in package.json instead, never force someone to have a global dependency (outside of system dependencies)

Reply:
I made this readme not only for those who know how to add local paths to their `PATH` env variable, but also for humans who just want to get up and running quickly.

Besides, I am not forcing anyone to do that. I just described it.